app/views/hosts/_form.html.erb

```diff
@@ -131,6 +136,9 @@
         <%= textarea_f f, :comment, :help_block => _("Additional information about this host"), :size => "col-md-8", :rows => "3", :class => "no-stretch" %>
       </div>
 
+      <% @tabs.values.each do |tab| %>
+        <%= tab %>
```

Comment:
I think the only local worth passing is "f" (the form builder object), other than that, everything smells fishy to me, @host or whatever should be an instance variable, not a local one (that is, in the tabs of the hosts#edit page)